A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/Clock/A320_Neo_Clock.css

```diff
@@ -51,34 +51,55 @@ a320-neo-clock-element {
   overflow: hidden; }
   a320-neo-clock-element #Mainframe {
     position: absolute;
+    /* top: -4%; */
     top: -4%;
     bottom: 0;
     left: 2%;
-    right: 0; }
+    right: 0; 
+    background-color: rgba(5, 40, 80, 0.623);
+    color: rgba(155, 148, 148, 0.911);}
     a320-neo-clock-element #Mainframe #TopSelector {
       margin-top: 0%;
+      /* width: 100%;
+      height: 35%; */
       width: 100%;
       height: 35%;
       display: flex;
       align-items: center;
       justify-content: center; }
       a320-neo-clock-element #Mainframe #TopSelector #TopSelectorValue {
         font-family: Alarm-Clock;
-        font-size: 65px;
+        font-size: 81px;
         text-align: center;
-        color: white; }
+        color: rgba(155, 148, 148, 0.911); }
     a320-neo-clock-element #Mainframe #MiddleSelector {
       margin-top: 0%;
-      width: 100%;
+      margin-left: 0%;
+      width: 80%;
       height: 35%;
-      display: flex;
+      display: flex; 
       align-items: center;
-      justify-content: center; }
+      justify-content: center; 
+      }
       a320-neo-clock-element #Mainframe #MiddleSelector #MiddleSelectorValue {
         font-family: Alarm-Clock;
-        font-size: 65px;
-        text-align: center;
+        font-size: 81px;
+        margin-left: 2%;
+        margin-right: 5%;
+        width: 95%;
+        text-align: right;
+        /* font-kerning: none; */
```

Comment:
Please remove comments ;)

Reply:
I will leave this one for now as a placeholder. This waiting for a new monospaced font to be creted for the chrono. #fonts guys are on it as it seems

Reply:
> I will leave this one for now as a placeholder. This waiting for a new monospaced font to be created for the chrono. #fonts guys are on it as it seems
> 

Okay perfect :)